### PR TITLE
[Config] Refactor & fix DB, DBPP, DRRG configs

### PR DESCRIPTION
**Pull request by gaotongxiao**
merge after #1188 

## Modifications

1. Refactor base dataset configs
2. Add `textdet_default_runtime.py` to `_base_`
3. Refactor configs of dbnet, dbnetpp and drrg:
    - Extract `model` and `*_pipeline` to `_base_{model_name}.py`
    - Use the dataset in `dataloader`

### CODE REVIEW

File: configs/textdet/dbnet/_base_dbnet_r18_fpnc.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        module_loss=dict(type='DBModuleLoss'),
+        postprocessor=dict(type='DBPostprocessor', text_repr_type='quad')),
+    data_preprocessor=dict(
+        type='TextDetDataPreprocessor',
+        mean=[123.675, 116.28, 103.53],
+        std=[58.395, 57.12, 57.375],
+        bgr_to_rgb=True,
+        pad_size_divisor=32))
+
+train_pipeline_r18 = [
```
Comment (Harold-lkk) [inline configs/textdet/dbnet/_base_dbnet_r18_fpnc.py:29]: maybe rename it to train_pipeline. Same for others

File: configs/textdet/dbnet/dbnet_r18_fpnc_1200e_icdar2015.py [hunk 2]
```diff
@@ -1,77 +1,32 @@
...
     num_workers=8,
-    persistent_workers=False,
+    persistent_workers=True,
     sampler=dict(type='DefaultSampler', shuffle=True),
     dataset=dict(
-        type='ConcatDataset', datasets=train_list,
-        pipeline=train_pipeline_r18))
+        type='ConcatDataset',
+        datasets=train_list,
+        pipeline=_base_.train_pipeline))
```
Comment (Harold-lkk) [inline configs/textdet/dbnet/dbnet_r18_fpnc_1200e_icdar2015.py:21]: do we need 'ConcatDataset'?

```
ic15_det_train = _base_.ic15_det_train
ic15_det_train.pipeline = train_pipeline
train_dataloader = dict(
    batch_size=16,
    num_workers=8,
    persistent_workers=True,
    sampler=dict(type='DefaultSampler', shuffle=True),
    dataset=ic15_det_train)
```

Comment (gaotongxiao) [reply configs/textdet/dbnet/dbnet_r18_fpnc_1200e_icdar2015.py:21]: We can leave `ConcatDataset` as a convention since it makes things eaiser

Comment (Harold-lkk) [reply configs/textdet/dbnet/dbnet_r18_fpnc_1200e_icdar2015.py:21]: It's a little difficult to debug and  use some features in OCRDataset like `indices`

File: configs/textdet/dbnetpp/_base_dbnetpp_r50dcnv2_fpnc.py [hunk 3]
```diff
@@ -56,7 +56,7 @@
         meta_keys=('img_path', 'ori_shape', 'img_shape'))
 ]
 
-test_pipeline_4068_1024 = [
+test_pipeline = [
```
Comment (Harold-lkk) [inline configs/textdet/dbnetpp/_base_dbnetpp_r50dcnv2_fpnc.py:59]: remove 'instances' in PackTextDetInputs, same for others

**Review by Harold-lkk — APPROVED** [review]

**Pull request merged**